artifacts/deployment/02-sample-apiserver-certs.yaml

```diff
@@ -22,8 +22,10 @@ kind: Secret
 metadata:
   name: etcd-client-cert
   namespace: k8s-sample-apiserver
-type: Opaque
+type: kubernetes.io/tls
 data:
+  tls.crt: ""
```

Comment:
When using `kubernetes.io/tls` Secrets, you must specify `tls.crt` and `tls.key`. Note that type is immutable, so it's not possible to update it without recreating the Secret.

As we don't have a RBAC role to delete and create a new Secret, and we don't want to extend required role set at all, maybe the best solution is to just specify empty `tls.crt` and `tls.key`.

Reply:
what does this mean for picking up rotated keys/certs in the apiserver?

Reply:
Relevant issues #49 and #50.

Reply:
For reference: those secrets are remounted in the apiserver pod.